main.sh

```diff
@@ -211,10 +211,11 @@ run_cart_test(){
 run_mdtest(){
     echo -e "\nCMD: Starting MDTEST...\n"
     for i in "${IOR_PROC_PER_CLIENT[@]}"; do
+	no_of_ps=$(($DAOS_CLIENTS * $i))
 	echo
-        cmd="orterun --timeout 600 $PSM2_CLIENT_PARAM -np $i --map-by node
+        cmd="orterun --timeout 600 $PSM2_CLIENT_PARAM $VERBS_CLIENT_PARAM -np $no_of_ps --map-by node
```

Comment:
So no other parameter needed for orterun command ?

Reply:
The current one's are working as of now. If something changes in future we can add.